### DISCUSSION TITLE
Switch from monitor to ip mode - run with external access point

### DISCUSSION THREAD

**Discussion by SummerSeaSun** [svpcom/wfb-ng#104] [Questions]
Dears,
I'm reading wfb_tx and wfb_rx code, and wondering if there is a way to run it in a LAN:

`svpcom_wifibroadcast/wfb_tx -k 1 -n 1 -K /home/pi/cameracontrol/IPCamera/svpcom_wifibroadcast/tx.key -u {UDP_PORT_OUT} -p 92 -B 20 -M 0 {interface}
`
i.e. replacing the {interface} with an IP address.

I've connected an ethernet cable to an external router, the ehternet interface cannot be set in monitor mode. What can I do?

Thanks in advance

**Comment by Mysnomer0** [comment]
My understanding is that only some chipsets support monitor mode. That's part of why we have specific hardware and modified drivers to make wifibroadcast work. Though, I do think what you're trying to do running on LAN with routers is really cool. Hope that helps! 

Thanks,

Milan
